plist.go

```diff
@@ -163,8 +163,7 @@ func makeVariableAction(token *token, varUUID *string) {
 		dataType: Text,
 		value:    token.ident,
 	}
-	if token.typeof != Var &&
-		(token.typeof == AddTo || token.typeof == SubFrom || token.typeof == MultiplyBy || token.typeof == DivideBy) &&
+	if (token.typeof == AddTo || token.typeof == SubFrom || token.typeof == MultiplyBy || token.typeof == DivideBy) &&
```

Comment:
Since the first condition was taken out, please remove the redundant parenthesis.